source/_integrations/mazda.markdown

```diff
@@ -55,6 +55,50 @@ Displays the current door lock status of the vehicle, and locks/unlocks the door
     The "Automatic Re-Lock" feature will automatically re-lock the doors if they are not opened shortly after being unlocked. This applies regardless of whether you are using the key, or unlocking the doors remotely using Home Assistant or the MyMazda app.
 </div>
 
+## Services
+
+This integration offers several services for interacting with Mazda vehicles. All of the services require the following input parameter:
+
+| Parameter | YAML ID | Description |
+| --------- | ------- | ----------- |
+| Vehicle | `device_id` | The vehicle to execute the service for. In YAML mode, this is the device ID of the vehicle. |
+
+### Start engine
+
+Starts the vehicle engine. The vehicle engine can only be remotely started 2 consecutive times. To reset this counter, the vehicle must be driven.
```

Comment:
The service name is missing (applies to all services).

Please check other services documentation for a reference.